fragments/quick_articles.php

```diff
@@ -74,6 +74,10 @@
 			{
 				$status_css = " qn_offline";
 			}
+			if ($data['status']== '2')
```

Comment:
Ich denke man sollte hier prüfen ob das accessdenied addon installiert/aktiviert ist

Welche status es gibt und was sie bedeuten ist sehr von addons abhängig. Die „2“ kann mit anderen addons/workflows was ganz anderes bedeuten

Reply:
[Überlegung] Denkbar wäre eine Einstellmöglichkeit in den Einstellungen des Addons, in Abhängigkeit der installierten Addons? [/Überlegung]
Klingt gut. Darüber sollte die Status-Ziffer geziehlt angepasst werden können (inkl. farbliche Darstellung).

Reply:
Es gibt keine Einstellungen für das AddOn, nur User-Settings. Das möchte ich auch so beibehalten.

Reply:
Man könnte hier einen ep setzen und die AddOns können sich hier einklinken.